versions.props

```diff
@@ -49,6 +49,7 @@ org.apache.httpcomponents:httpmime=4.5.14
 org.apache.kerby:*=1.0.1
 org.apache.logging.log4j:*=2.21.0
 org.apache.lucene:*=9.9.2
```

Comment:
Temporarily within this pull request (pre-merge) we could change this to a Lucene 10 prerelease based on the _"Update Lucene prerelease"_ steps in https://github.com/apache/solr/blob/main/help/dependencies.txt ...

... might be worth waiting though until Lucene 9.10 is out and https://issues.apache.org/jira/browse/SOLR-17157 has upgraded Solr to use it i.e. then `solr/main` will be closer to `lucene/main` than it is right now.